components/src/core/hero/hero.tsx

```diff
@@ -179,6 +193,46 @@ export const Hero: React.FC<HeroProps> = (props) => {
             );
         }
     }, [ValueIconClass]);
+    useEffect(() => {
+        if (fontSize) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                `Property 'fontSize' in Hero component has been deprecated and will be removed in version 6.0.0. You should update to use the 'channelValueProps' prop instead.`
+            );
+        }
+    }, [fontSize]);
+    useEffect(() => {
+        if (value) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                `Property 'value' in Hero component has been deprecated and will be removed in version 6.0.0. You should update to use the 'channelValueProps' prop instead.`
+            );
+        }
+    }, [value]);
+    useEffect(() => {
+        if (valueIcon) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                `Property 'valueIcon' in Hero component has been deprecated and will be removed in version 6.0.0. You should update to use the 'channelValueProps' prop instead.`
+            );
+        }
+    }, [valueIcon]);
+    useEffect(() => {
+        if (valueColor) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                `Property 'valueColor' in Hero component has been deprecated and will be removed in version 6.0.0. You should update to use the 'channelValueProps' prop instead.`
+            );
+        }
+    }, [valueColor]);
+    useEffect(() => {
+        if (units) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                `Property 'units' in Hero component has been deprecated and will be removed in version 6.0.0. You should update to use the 'channelValueProps' prop instead.`
+            );
+        }
+    }, [units]);
```

Comment:
You have the deprecation warnings in here and they look good, but you also need to add the 'forward compatibility' so they can use the ChannelValueProps or the individual props in 5.4. That is, you'll need to include the changes that you made for 6.0.0 to introduce the ChannelValueProps prop, and then add some logic to either use the ChannelValueProps (if they're using it) or the individual props here in 5.4.

A very simple example for doing this with Icons can be seen in a few components like the EmptyState (https://github.com/pxblue/react-native-component-library/blob/dev/components/src/core/empty-state/empty-state.tsx#L108)

Essentially, you'll do something like:

```
const ChannelProps = props.ChannelValueProps || {icon: props.valueIcon, fontSize: props.fontSize, ...}
```
and then do the prop pass through just as you have it set up in 6.0.0.

Reply:
You'll also need to update the docs to show that the old props are deprecated and what the new prop is.